ckb/src/main/java/org/nervos/ckb/utils/EpochParser.java

```diff
@@ -14,6 +14,11 @@ public static EpochParams parse(String hexEpoch) {
   }
 
   public static String parse(long length, long index, long number) {
```

Comment:
this seems to be generating epoch not parse epoch?

Reply:
It's interesting and I am not sure `generate` is fine, so I use `parse` to handle. 
I rename it to `generate`.

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/293/commits/ca45791f6ce1bfb72fe37dd149750fc14311c7b7

Reply:
according to https://github.com/nervosnetwork/ckb/pull/1585, I think it's more appropriate to call `generate`